v2/concordium/service.proto

```diff
@@ -139,4 +139,13 @@ service Queries {
   // for API consistency.
   // The stream will end when all the non-finalized transaction hashes have been returned.
   rpc GetAccountNonFinalizedTransactions (AccountAddress) returns (stream TransactionHash);
+
+  // Get a list of banned peers.
+  rpc GetBannedPeers(Empty) returns (BannedPeers) {}
+
+  // Ban a the given peer.
+  rpc BanPeer(PeerToBan) returns (BooleanResponse) {}
```

Comment:
The response is not described here. If we keep it a boolean then we should just use `bool`.

Otherwise either return `Empty` and signal errors using the grpc error signals. If there is a meaningful  response we should have an `enum` or `oneof`.